### ISSUE TITLE
Autocomplete for method overwrites should add default args

### ISSUE THREAD

**Issue by cdce8p** [microsoft/pylance-release#869]
## Environment data

-   Language Server version: 2021.1.2
-   OS and version: darwin x64
-   Python version: 3.9.1 
-   python.analysis.indexing: true
-   python.analysis.typeCheckingMode: off

## Expected behaviour

When using autocomplete for an overwritten function, pylance should add default values.
Otherwise one might assume this to be the correct function signature which could cause potential issues.

If pylance doesn't know the correct default value, it would already be helpful to only add `= ...` as in stub files. That way you would notice it directly and know that you still need to change some things.

## Actual behaviour

Pylance only adds function parameter and their type hints.

## Logs
--

## Code Snippet / Additional information

```py
import argparse
from typing import Optional, Text

class CustomFormatter(argparse.HelpFormatter):
    def __init__  # use autocomplete here

## Result
class CustomFormatter(argparse.HelpFormatter):
    def __init__(self, prog: Text, indent_increment: int, max_help_position: int, width: Optional[int]) -> None:
        super().__init__(prog, indent_increment=indent_increment, max_help_position=max_help_position, width=width)

# The default values would be
indent_increment = 2
max_help_position = 24
width = None
```

**Comment by jakebailey** [comment]
The next release will generate default values to ensure that the generated signatures aren't invalid (which is the case now).

**Comment by jakebailey** [comment]
This issue has been fixed in version 2021.9.2, which we've just released. You can find the changelog here: https://github.com/microsoft/pylance-release/blob/main/CHANGELOG.md#202192-16-september-2021
